README.md

```diff
@@ -59,7 +59,9 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+var weather; declares an undefined variable called weather.
```

Comment:
All these statements are true, but you didn't mention with the expressions evaluate to. As it happens, the results are `undefined`, `"sunny"`, and `true`.